.github/workflows/apk-s3-distribute.yml

```diff
@@ -4,6 +4,10 @@ on:
   release:
     types: [released]
 
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
```

Comment:
We probably don't need to apply these to the distribute workflows, right?

Reply:
As far as we only want the last compilation, we can add it.
If there is a new commit that arrives to this branch, the current job is not needed anymore and we want the new one to be run ASAP

Reply:
For example with releases to Sonatype, we might create a repo and start uploading files to it. If we cancel the job and the process gets killed off, we'd leave non-closed staging repos that have some random amount of release files in them, wouldn't we?

Plus it seems like it wouldn't be a huge win to cancel something like a snapshot release. The big gains would come from pretty much just the PR workflows (maybe the `develop` builds, but those merges can't really happen quickly enough to cancel each other because there has to be at least one PR workflow run inbetween anyway). Isn't it enough to add cancellation for that workflow?

Reply:
They shouldn't go so quickly, so, it shouldn't be canceled never.
BTW, I am going to remove the one from the publish-snapshot, because keep the repository opened could gives us some issues (following snapshots not published, Manually close it....)
For the rest of them I think it should be fine. Maybe it is never canceled, but if it is canceled no inconsistent data is generated

Reply:
I'd prefer adding it only where we see that it actually makes sense and provides a benefit. Otherwise simpler > better in this case, isn't it?

I can imagine similar weird leftover things in other cases, in Firebase app distribution as well for example.